### PR TITLE
2022q4 portmgr

### PR DESCRIPTION
**Pull request by rene0**
Just-in-time :)

### CODE REVIEW

File: 2022q4/portmgr.adoc [hunk 1]
```diff
@@ -0,0 +1,67 @@
+=== Ports Collection
+
+Links: +
+link:https://www.FreeBSD.org/ports/[About FreeBSD Ports] URL:link:https://www.FreeBSD.org/ports/[https://www.FreeBSD.org/ports/] +
```
Comment (pauamma) [inline 2022q4/portmgr.adoc:4]: s/URL:/URL: /

File: 2022q4/portmgr.adoc [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+We welcomed pizzamig@ as a new official member after a succesful lurking period.
+We also welcomed three new lurkers: bofh@, ler@, and ygy@.
+
+Portmgr split itself up into portmgr and pkgmgr.
+The new pkgmgr team, currently consisting of anoine@ and bdrewery@, is responsible for building packages and maintaining the package building cluster.
+
+Four new USES were introduced:
+* llvm to canocalize ports dependencies on LLVM
+* luajit to select a LuaJIT runtime
+* octave to help ports to depend on Octave and Octave-Forge
```
Comment (pauamma) [inline 2022q4/portmgr.adoc:31]: s/to depend/depend/ (or maybe "that depend").

File: 2022q4/portmgr.adoc [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+The following default versions were bumped:
+* Firebird to 3.0
+* GCC to 12
+* Lazarus to 2.2.4
+* Lua to 5.4
+* PHP to 8.1
+* Samba to 4.13
+* Varnish to 6
+* LuaJIT is new and at luajit-devel or luajit-openresty for PowerPC64
```
Comment (pauamma) [inline 2022q4/portmgr.adoc:42]: "at"?

File: 2022q4/portmgr.adoc [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+* gcc 12
+* KDE Plasma 5.24.7, Frameworks 5.101.0, Applications 22.12.0
+* Qt 5.15.7 and 6.4.1
+* Rust 1.66.0
+* SDL 2.26.1
+* Sway 1.8
+* wlroots 0.16.1
+* Wine 7.0.1
+
+The exp-run report are available again.
```
Comment (pauamma) [inline 2022q4/portmgr.adoc:62]: s/report/reports/

File: 2022q4/portmgr.adoc (hunk 2 above)
Comment (rene0) [reply 2022q4/portmgr.adoc]: "depend"  it is

File: 2022q4/portmgr.adoc (hunk 3 above)
Comment (rene0) [reply 2022q4/portmgr.adoc]: Yeah, I rephrased this a bit.

**Review by pauamma — APPROVED** [review]

**Pull request merged**